### PR TITLE
pom.xml cleanup - consistent formatting, remove unused profiles, and …

### PR DESCRIPTION
**Pull request by ashitsalesforce**
…dependencies

continued pom.xml cleanup - consistent formatting, remove unused profiles, and dependencies

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -468,25 +390,7 @@
           <OSType>mac</OSType>
       </properties>
     </profile>
-    <profile>
```
Comment (jjangsfdc) [inline pom.xml:471]: win32 is old! Frankly, I don't even know if it will work in win 32bits.

File: pom.xml [hunk 2]
```diff
@@ -398,57 +371,6 @@
...
-      </activation>
-      <dependencies>
-        <dependency>
-          <groupId>org.eclipse.swt</groupId>
-          <artifactId>org.eclipse.swt.gtk.linux.x86</artifactId>
-          <version>4.5.1</version>
-        </dependency>
-      </dependencies>
-    </profile>
-    <profile>
```
Comment (jjangsfdc) [inline pom.xml:417]: Removing 32 bit OS profiles - yes. I know we don't officially support Linux but there might be users who are dependent on `linux64`, maybe not. We can remove it or not, your call.

**Review by jjangsfdc — APPROVED** [review]

**Comment by ashitsalesforce** [comment]
Thanks Jae. I decided to remove all profiles that we do not maintain/build
with to avoid confusing build failures for the community members in case
they try to build with those profiles.

Ashit

File: pom.xml (hunk 1 above)
Comment (ashitsalesforce) [reply pom.xml:471]: Exactly. Having such obsolete profiles is confusing at best, frustrating if the community members try to use them.

**Pull request merged**